src/loaders/ObjectLoader.js

```diff
@@ -610,6 +617,94 @@ THREE.ObjectLoader.prototype = {
 
 			}
 
+			/*
+			 * Now SkinnedMesh could have two bone sets in its children,
+			 * one is made from geometry.bones in SkinnedMesh constroctor
+			 * and another one is made from data.children in this method.
+			 * These two bone sets correspond to the same bones.
+			 * The former would represent a default stand pose while
+			 * the latter would represent a pose updated by skinning.
+			 * To let SkinnedMesh be updated pose, here this logic copies
+			 * the latter one to the former one then removes the latter one.
+			 * If in the latter one there're objects which don't correspond to
+			 * the ones in the former one, they'll be moved to the former one.
+			 */
```

Comment:
I think that we still convert node-based bones into skeleton bones during load time if I am not mistaken.  If we were to only use skeleton-bones by default, we could likely speed up load time.  I could see it useful to have a function to convert skeleton-bones to nodes and vice versa, but I would only keep one representation at a time to avoid issues -- although as I said modifying the bones in a game engine will likely invalidate all saved animations for that character -- which is likely a bad thing.